docs/components/topnav/test.html

```diff
@@ -7,7 +7,7 @@
 <div id="vue-topNavDemo">
```

Comment:
@danielmdesigns, please see this page for a full-page demo of `Top Navigation`:

https://deploy-preview-784--helix-ui.netlify.app/components/topnav/test.html

Reply:
@100stacks Here are a few things that I noticed

- The logo area does not have a hover state
- nav-items need ~3px-4px of padding added to each side (left & right)
- I can't make a selection from the product selector, so not sure if the "active" state is available/correct
- The label "Account Number" from the dropdown should not be a hoverable/clickable link

Reply:
@danielmdesigns, thanks for the feedback.  Please see below:

1. Please verify hover state for the logo area.  It appears to work in the PR preview.
1. Please add a screenshot of which nav-items need the padding.
1. This is a demo example, so we don't have it "wired up" to any logic.
1. Ok, good catch.  We'll fix this.

Reply:
![nav-padding](https://user-images.githubusercontent.com/5085619/98729001-d8237600-235f-11eb-876e-0cc1f4a1e2f1.png)

Thanks for the clarification... Take a look at the attachment of the nav-item. They all appear to be ~9px of left/right padding and it should be 12px. Let me know if I am missing something but it's a little tight side-to-side